torch/csrc/jit/passes/quantization.cpp

```diff
@@ -886,5 +886,131 @@ graph(%self, %scale, %zero_point, %dtype):
+      }
+      for (Block* subblock : n->blocks()) {
+        blocks_to_visit.push(subblock);
+      }
+    }
+  }
+
+  auto patterns = {linear_with_quant};
+  for (const auto& pattern: patterns) {
+    Graph pattern_graph;
+    std::unordered_map<std::string, Value*> vmap;
+    script::parseIR(pattern, &pattern_graph, vmap);
+    const auto& matches = findPatternMatches(pattern_graph, *graph);
+    TORCH_CHECK(
+        matches.size() <= 1, "We only support at most one match right now");
+    for (const auto& match : matches) {
+      const auto& match_vmap = match.values_map;
+      auto w_scale = toIValue(match_vmap.at(vmap.at("w_scale"))).value().toDouble();
+    auto w_zero_point =
+      toIValue(match_vmap.at(vmap.at("w_zero_point"))).value().toInt();
+    auto w_dtype =
+        toIValue(match_vmap.at(vmap.at("w_dtype"))).value().toScalarType();
+      auto w = module.get_parameter("weight").variable_data();
+      auto w_quant = at::quantize_per_tensor(w, w_scale, w_zero_point, w_dtype);
+      auto b = module.get_parameter("bias").variable_data();
+      auto wrapper = wrapper_module.clone();
+      auto set_weight_bias = wrapper.get_method("set_weight_bias");
+      set_weight_bias(std::vector<IValue>{IValue(w_quant), IValue(b)});
+      // TODO: we need to make sure this name is unique
+      module.register_module(
+          "_packed_linear_weight_bias",
+          wrapper
+      );
+
+      // Replace GetAttr on %self with PackedParams module
+      auto w_val = match_vmap.at(vmap.at("w"));
+      auto b_val = match_vmap.at(vmap.at("b"));
+      auto w_quant_val = match_vmap.at(vmap.at("w_quant"));
+
+      // m = self._packed_linear_weight_bias
+      Node* packed_params_module = graph->create(prim::GetAttr);
+      packed_params_module->addInput(graph->inputs()[0]);
+      packed_params_module->s_(attr::name, "_packed_linear_weight_bias");
+      packed_params_module->output()->setDebugName("m");
+      packed_params_module->output()->setType(wrapper.type());
+      packed_params_module->insertAfter(w_quant_val->node());
+
+      // _packed_params = m._packed_params
+      Node* packed_params = graph->create(prim::GetAttr);
+      packed_params->addInput(packed_params_module->output());
+      packed_params->s_(attr::name, "_packed_params");
+      packed_params->output()->setDebugName("_packed_params");
+      packed_params->insertAfter(packed_params_module);
+
+      // weight_bias = m._weight_bias
+      Node* unpack_node = graph->create(prim::CallMethod);
+      unpack_node->s_(attr::name, "_weight_bias");
+      unpack_node->addInput(packed_params_module->output());
+      unpack_node->output()->setDebugName("weight_bias");
+      unpack_node->output()->setType(TupleType::create(std::vector<TypePtr>({TensorType::get(), OptionalType::create(TensorType::get())})));
+      unpack_node->insertAfter(packed_params);
+
+      WithInsertPoint ins(unpack_node);
```

Comment:
If we're using the insert point API, let's use it everywhere for consistency: please replace `insertAfter` with `insertNode`.